Source/DriveWorks.Sdk.Examples.CSharp/MyErrorStrings.cs

```diff
@@ -0,0 +1,27 @@
+﻿using DriveWorks.Extensibility;
+using Titan.Rules.Execution;
+
+namespace DriveWorks.Sdk.Examples.CSharp
+{
+    public class ExampleErrorStrings : SharedProjectExtender
+    {
+        private const string EXAMPLE_ERROR_STRINGS = "#FINDUSERID! User ID invalid.";
+        /// <remarks>
+        /// UDF validates User ID input.
+        /// If User ID is 3 characters in length function will output successful message otherwise an error. 
+        /// </remarks>   
+        [Udf(true)]
+        [FunctionInfo("ExampleErrorString.", "SDK-Starter-Examples Plugin")]
```

Comment:
Lets improve the UDF description.
"This function returns an error string if the user ID supplied is not 3 characters in length. Error strings can be caught in rules using IfErrorString() or IsErrorString()"